src/database/Mongo.ts

```diff
@@ -83,4 +83,20 @@ export class Mongo {
 			});
 		}
 	}
+
+	public async updateOne(id: number, update: object): Promise<any> {
```

Comment:
We should set a type here, I think it might be `Promise<FindAndModifyWriteOpResultObject>` but I'm not 100%. Mongo has built-in types to define their responses tho, I'm just not sure what the response from updateOne looks like